.gitignore

```diff
@@ -101,7 +101,8 @@ tmp
 wheelhouse/
 *.whl
 
-.pytest_cache
```

Comment:
Note: this `.pytest_cache` entry was a duplicate (we already have it in the testing section higher in this file).